tests/parshaping-simple.sil

```diff
@@ -0,0 +1,89 @@
+\begin[papersize=6in x 9in, class=book]{document}
+\language[main=en]
+\hyphenator:add-exceptions[lang=en]{iso-pe-ri-me-t-ric}% No idea where that weird word should really break.
+
+\script{
+-- A pre-computed circle for the 6in page width with book class page masters, and default
+-- Gentium Plus font. If any of these changes, the test may break.
+local circle = {
+  { width = SILE.measurement("36.779999477581pt"), indent=SILE.measurement("149.44200026121pt") },
+  { width = SILE.measurement("89.148920336526pt"), indent=SILE.measurement("123.25753983174pt") },
+  { width = SILE.measurement("116.65835432049pt"), indent=SILE.measurement("109.50282283975pt") },
+  { width = SILE.measurement("135.41673924727pt"), indent=SILE.measurement("100.12363037636pt") },
+  { width = SILE.measurement("148.77195603998pt"), indent=SILE.measurement("93.446021980012pt") },
+  { width = SILE.measurement("158.09926166671pt"), indent=SILE.measurement("88.782369166646pt") },
+  { width = SILE.measurement("164.08698356407pt"), indent=SILE.measurement("85.788508217965pt") },
+  { width = SILE.measurement("167.09452358097pt"), indent=SILE.measurement("84.284738209513pt") },
+  { width = SILE.measurement("167.28269918420pt"), indent=SILE.measurement("84.190650407902pt") },
+  { width = SILE.measurement("164.66117660805pt"), indent=SILE.measurement("85.501411695975pt") },
+  { width = SILE.measurement("159.09112080046pt"), indent=SILE.measurement("88.286439599772pt") },
+  { width = SILE.measurement("150.24495450145pt"), indent=SILE.measurement("92.709522749275pt") },
+  { width = SILE.measurement("137.49177425846pt"), indent=SILE.measurement("99.086112870769pt") },
+  { width = SILE.measurement("119.58808312009pt"), indent=SILE.measurement("108.03795843996pt") },
+  { width = SILE.measurement("93.624309129279pt"), indent=SILE.measurement("121.01984543536pt") },
+  { width = SILE.measurement("47.922780548840pt"), indent=SILE.measurement("143.87060972558pt") },
+  { width = SILE.measurement("167.54343340503pt"), indent=SILE.measurement("84.060283297483pt") }
+}
```

Comment:
This is scary to put in a script, surely with Lua at your disposal you can calculate something as simple as points along a circle?

Reply:
My assumption was that in a test script, one wants repeatable values (known in advance). It doesn't even needs to be a circle here.

Reply:
Should avoid relying on the book class page masters though, an use only plain.

Reply:
Hard coded values in tests are fine. Using *plain* instead of *book* for tests is almost always preferred except where the thing being tested stands any reasonably chance of being affected by the class. As with folios and other things, the less things that get stuffed into unrelated tests the better.